site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -45,24 +45,77 @@ postman files later, you can just create a new environment again to work with
 Select the workspace menu at the top of the Postman interface, and create a new one
 called 'Egeria dojo':
 
-![Creating a new Environment 1](postman-workspace1.png)
-![Creating a new Environment 2](postman-workspace2.png)
-![Creating a new Environment 3](postman-workspace3.png)
+![Creating a new Environment 1](postman-create-workspace1.png)
```

Comment:
I wonder if it would be easier to read if it were done like the installing Intellij where each screen shot it sin a folding section with a description, it is then more obvious what each screen shot means. I guess this would be easier for a screen reader which could not see the screen shot as well.

Reply:
I've given more detailed instructions, and included the screenshot in a collapsible section.